tests/e2e/split-two-columns.spec.js

```diff
@@ -0,0 +1,62 @@
+import {test, expect} from '@playwright/test';
+import {login} from './tools/lib/login';
+
+test('Create and check check split two column block', async ({page, context}) => {
+  await page.goto('./');
+  await login(page, context);
+
+  await page.goto('./wp-admin/post-new.php?post_type=page');
+
+  await page.getByRole('button', {name: 'Close'}).click();
+  const secondCloseBtn = page.getByRole('button', {name: 'Close'});
+  if (await secondCloseBtn.isVisible()) {
+    await secondCloseBtn.click();
+  }
+
+  // Adding Split Two Column Block
+  await page.locator('.block-editor-block-list__layout').click();
+  await page.locator('p.is-selected.wp-block-paragraph').fill('/split');
```

Comment:
I've had some issues with `fill` recently, it was not triggering the popover in the editor in Chrome, if you use [`type`](https://playwright.dev/docs/api/class-locator#locator-type) it seems more reliable